Plugin Install Errors

First install of the plugin makes the big :cry: 

[Stacktrace](https://sentry.io/share/issue/5ae697b5f2a342febbb2936eaca0f680/)


Additional Info | None
-- | --
App Name | IntelliJ IDEA 2020.3.1 (Community Edition)
Build Info | Build #IC-203.6682.168, built on December 29, 2020
Cores | 12
Current LAF | KonoSuba: Aqua
GC | ParNew,ConcurrentMarkSweep
JRE | Runtime version: 11.0.9.1+11-b1145.63 amd64
Memory | 4012
Non-Bundled Plugins | IdeaVIM,com.jetbrains.ChooseRuntime,io.acari.DDLCTheme,io.acari.normandy.progress.bar,org.nik.presentation-assistant,org.jetbrains.kotlin,io.unthrottled.amii
Plugin Config | {   "memeVolume": 75,   "soundEnabled": true,   "memeDisplayModeValue": "TIMED",   "memeDisplayAnchorValue": "TOP_RIGHT",   "memeDisplayInvulnerabilityDuration": 3,   "memeDisplayTimedDuration": 40,   "userId": "",   "version": "",   "allowedExitCodes": "0,130",   "idleTimeoutInMinutes": 5,   "allowFrustration": true,   "eventsBeforeFrustration": 5,   "probabilityOfFrustration": 75,   "preferredCharacters": "",   "logSearchTerms": "",   "logSearchIgnoreCase": true,   "preferredGenders": 7,   "enabledEvents": 253,   "showMood": true }
Registry | <invalid>
System Info | Linux 5.4.0-58-generic
VM | VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

Part of the logs:

<details>
<summary>idea.log</summary>

```
2021-01-02 18:05:14,940 [ 207911]   INFO - ed.amii.integrations.RestTools - Attempting to download remote asset: https://jetbrains.assets.unthrottled.io/amii/sentry-dsn.txt
2021-01-02 18:05:15,044 [ 208015]   INFO - ed.amii.integrations.RestTools - Asset has responded for remote asset: https://jetbrains.assets.unthrottled.io/amii/sentry-dsn.txt with status code 200
2021-01-02 18:05:15,089 [ 208060]   INFO - .diagnostic.PerformanceWatcher - Pushing properties took 3ms; general responsiveness: ok; EDT responsiveness: ok
2021-01-02 18:05:15,460 [ 208431]   INFO - rationStore.ComponentStoreImpl - Saving appExportableFileTemplateSettings took 19 ms
2021-01-02 18:05:15,689 [ 208660]   INFO - mponents.impl.stores.StoreUtil - saveProjectsAndApp took 17505 ms
2021-01-02 18:05:15,737 [ 208708]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/addons_list-3.xml
2021-01-02 18:05:16,011 [ 208982]   INFO - dea.updater.SdkComponentSource - File /home/alex/.android/repositories.cfg could not be loaded.
2021-01-02 18:05:16,011 [ 208982]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-automotive/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-wear-cn/sys-img2-1.xml
2021-01-02 18:05:16,011 [ 208982]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-wear/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google_apis/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google_apis_playstore/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/addon2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-tv/sys-img2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/glass/addon2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/extras/intel/addon2-1.xml
2021-01-02 18:05:16,012 [ 208983]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/repository2-1.xml
2021-01-02 18:05:16,838 [ 209809]   INFO - .diagnostic.PerformanceWatcher - Indexable file iteration took 1749ms; general responsiveness: ok; EDT responsiveness: 1/2 sluggish
2021-01-02 18:05:16,840 [ 209811]   INFO - indexing.UnindexedFilesUpdater - Unindexed files update started: 0 files to index
2021-01-02 18:05:28,606 [ 221577]  ERROR - openapi.options.ConfigurableEP - Cannot create configurable
com.intellij.diagnostic.PluginException: Cannot create class io.unthrottled.amii.config.ui.PluginSettingsUI [Plugin: io.unthrottled.amii]
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:725)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateExtensionWithPicoContainerOnlyIfNeeded(ComponentManagerImpl.kt:754)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:425)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:332)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.createConfigurable(ConfigurableWrapper.java:45)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:119)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:94)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:140)
	at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$new$0(GotoActionModel.java:93)
	at com.intellij.openapi.util.VolatileNotNullLazyValue$1.compute(VolatileNotNullLazyValue.java:56)
	at com.intellij.openapi.util.VolatileNotNullLazyValue.getValue(VolatileNotNullLazyValue.java:36)
	at com.intellij.ide.util.gotoByName.GotoActionModel.getConfigurablesNames(GotoActionModel.java:378)
	at com.intellij.ide.util.gotoByName.GotoActionItemProvider.processOptions(GotoActionItemProvider.java:176)
	at com.intellij.ide.util.gotoByName.GotoActionItemProvider.filterElements(GotoActionItemProvider.java:126)
	at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.fetchWeightedElements(ActionSearchEverywhereContributor.java:96)
	at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:179)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)
	at com.intellij.util.ConcurrencyUtil.lambda$underThreadNameRunnable$3(ConcurrencyUtil.java:196)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:270)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:684)
	... 28 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class io.unthrottled.amii.assets.VisualContentManager
	at io.unthrottled.amii.assets.VisualEntityRepository.createIndex(VisualEntityRepository.kt:73)
	at io.unthrottled.amii.assets.VisualEntityRepository.<init>(VisualEntityRepository.kt:55)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:47)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:733)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:49)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:37)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:113)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:67)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:457)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:440)
	at io.unthrottled.amii.assets.VisualEntityRepository$Companion.getInstance(VisualEntityRepository.kt:12)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.getCharacterList(PreferredCharacterTree.kt:133)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.getCharacterList$default(PreferredCharacterTree.kt:132)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.initTree(PreferredCharacterTree.kt:65)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.<init>(PreferredCharacterTree.kt:299)
	at io.unthrottled.amii.config.ui.PreferredCharacterPanel.<init>(PreferredCharacterPanel.java:11)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.createUIComponents(PluginSettingsUI.java:105)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.$$$setupUI$$$(PluginSettingsUI.java)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.<init>(PluginSettingsUI.java:65)
	... 33 more
2021-01-02 18:05:28,607 [ 221578]  ERROR - openapi.options.ConfigurableEP - IntelliJ IDEA 2020.3.1  Build #IC-203.6682.168
2021-01-02 18:05:28,607 [ 221578]  ERROR - openapi.options.ConfigurableEP - JDK: 11.0.9.1; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2021-01-02 18:05:28,607 [ 221578]  ERROR - openapi.options.ConfigurableEP - OS: Linux
2021-01-02 18:05:28,607 [ 221578]  ERROR - openapi.options.ConfigurableEP - Plugin to blame: AMII version: 0.1.0
2021-01-02 18:05:28,607 [ 221578]  ERROR - openapi.options.ConfigurableEP - Last Action: GotoAction
2021-01-02 18:05:28,607 [ 221578]   WARN - openapi.options.ConfigurableEP - Bundle key missed for null
2021-01-02 18:05:28,607 [ 221578]  ERROR - options.ex.ConfigurableWrapper - Can't instantiate configurable for io.unthrottled.amii.config.ui.PluginSettingsUI
java.lang.Throwable: Can't instantiate configurable for io.unthrottled.amii.config.ui.PluginSettingsUI
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:159)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:121)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:94)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:140)
	at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$new$0(GotoActionModel.java:93)
	at com.intellij.openapi.util.VolatileNotNullLazyValue$1.compute(VolatileNotNullLazyValue.java:56)
	at com.intellij.openapi.util.VolatileNotNullLazyValue.getValue(VolatileNotNullLazyValue.java:36)
	at com.intellij.ide.util.gotoByName.GotoActionModel.getConfigurablesNames(GotoActionModel.java:378)
	at com.intellij.ide.util.gotoByName.GotoActionItemProvider.processOptions(GotoActionItemProvider.java:176)
	at com.intellij.ide.util.gotoByName.GotoActionItemProvider.filterElements(GotoActionItemProvider.java:126)
	at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.fetchWeightedElements(ActionSearchEverywhereContributor.java:96)
	at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:179)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)
	at com.intellij.util.ConcurrencyUtil.lambda$underThreadNameRunnable$3(ConcurrencyUtil.java:196)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:270)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
2021-01-02 18:05:28,608 [ 221579]  ERROR - options.ex.ConfigurableWrapper - IntelliJ IDEA 2020.3.1  Build #IC-203.6682.168
2021-01-02 18:05:28,608 [ 221579]  ERROR - options.ex.ConfigurableWrapper - JDK: 11.0.9.1; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2021-01-02 18:05:28,608 [ 221579]  ERROR - options.ex.ConfigurableWrapper - OS: Linux
2021-01-02 18:05:28,608 [ 221579]  ERROR - options.ex.ConfigurableWrapper - Last Action: GotoAction
2021-01-02 18:05:28,608 [ 221579]   WARN - openapi.options.ConfigurableEP - Bundle key missed for null
2021-01-02 18:05:40,228 [ 233199]   INFO - j.ide.actions.RevealFileAction - Timed out
2021-01-02 18:05:40,229 [ 233200]   INFO - j.ide.actions.RevealFileAction -
(org.gnome.Nautilus:71196): Gtk-WARNING **: 18:05:30.412: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
Nautilus-Share-Message: 18:05:30.531: Called "net usershare info" but it failed: Failed to execute child process “net” (No such file or directory)
```
</details

Another relevant thing

<details>
<summary>error stacktrace</summary>

```
Cannot create configurable

com.intellij.diagnostic.PluginException: Cannot create class io.unthrottled.amii.config.ui.PluginSettingsUI [Plugin: io.unthrottled.amii]
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:725)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateExtensionWithPicoContainerOnlyIfNeeded(ComponentManagerImpl.kt:754)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:425)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:332)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.createConfigurable(ConfigurableWrapper.java:45)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:119)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:94)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:140)
	at com.intellij.openapi.options.ex.Weighted.lambda$static$0(Weighted.java:19)
	at java.base/java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
	at java.base/java.util.TimSort.sort(TimSort.java:220)
	at java.base/java.util.Arrays.sort(Arrays.java:1515)
	at java.base/java.util.ArrayList.sort(ArrayList.java:1750)
	at com.intellij.openapi.options.ex.SortedConfigurableGroup.buildConfigurables(SortedConfigurableGroup.java:41)
	at com.intellij.openapi.options.SearchableConfigurable$Parent$Abstract.getConfigurables(SearchableConfigurable.java:96)
	at com.intellij.openapi.options.ex.EpBasedConfigurableGroupKt.collect(EpBasedConfigurableGroup.kt:152)
	at com.intellij.openapi.options.ex.EpBasedConfigurableGroupKt.access$collect(EpBasedConfigurableGroup.kt:1)
	at com.intellij.openapi.options.ex.EpBasedConfigurableGroup$createListener$1.run(EpBasedConfigurableGroup.kt:99)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl$4.extensionListChanged(ExtensionPointImpl.java:875)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.notifyListeners(ExtensionPointImpl.java:782)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.lambda$registerExtensions$17(ExtensionPointImpl.java:1022)
	at com.intellij.ide.plugins.DynamicPlugins.loadPluginDescriptor(DynamicPlugins.kt:839)
	at com.intellij.ide.plugins.DynamicPlugins.access$loadPluginDescriptor(DynamicPlugins.kt:93)
	at com.intellij.ide.plugins.DynamicPlugins$loadPlugin$2.run(DynamicPlugins.kt:763)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1000)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin(DynamicPlugins.kt:759)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin(DynamicPlugins.kt:739)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin$default(DynamicPlugins.kt:738)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin(DynamicPlugins.kt)
	at com.intellij.ide.plugins.PluginInstaller.installAndLoadDynamicPlugin(PluginInstaller.java:356)
	at com.intellij.ide.plugins.newui.MyPluginModel.lambda$apply$1(MyPluginModel.java:179)
	at com.intellij.ide.plugins.InstalledPluginsState.trackPluginInstallation(InstalledPluginsState.java:170)
	at com.intellij.ide.plugins.newui.MyPluginModel.apply(MyPluginModel.java:178)
	at com.intellij.ide.plugins.PluginManagerConfigurable.apply(PluginManagerConfigurable.java:1675)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.apply(ConfigurableWrapper.java:182)
	at com.intellij.openapi.options.newEditor.ConfigurableEditor.apply(ConfigurableEditor.java:315)
	at com.intellij.openapi.options.newEditor.SettingsEditor$5.apply(SettingsEditor.java:176)
	at com.intellij.openapi.options.newEditor.SettingsEditor.apply(SettingsEditor.java:364)
	at com.intellij.openapi.options.newEditor.SettingsDialog.applyAndClose(SettingsDialog.java:183)
	at com.intellij.openapi.options.newEditor.SettingsDialog.doOKAction(SettingsDialog.java:179)
	at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1896)
	at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1849)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:270)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6652)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3345)
	at java.desktop/java.awt.Component.processEvent(Component.java:6417)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:976)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:911)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:840)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:507)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1063)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:694)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:435)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1715)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1674)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.java:134)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.java:107)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.java:99)
	at com.intellij.ide.actions.ShowPluginManagerAction.actionPerformed(ShowPluginManagerAction.java:19)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:281)
	at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:104)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:216)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:24)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:199)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:324)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:85)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:134)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:47)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:190)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:976)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:501)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:684)
	... 126 more
Caused by: java.lang.ExceptionInInitializerError
	at io.unthrottled.amii.assets.VisualEntityRepository.createIndex(VisualEntityRepository.kt:73)
	at io.unthrottled.amii.assets.VisualEntityRepository.<init>(VisualEntityRepository.kt:55)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:47)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:733)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:49)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:37)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:113)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:67)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:457)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:440)
	at io.unthrottled.amii.assets.VisualEntityRepository$Companion.getInstance(VisualEntityRepository.kt:12)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.getCharacterList(PreferredCharacterTree.kt:133)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.getCharacterList$default(PreferredCharacterTree.kt:132)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.initTree(PreferredCharacterTree.kt:65)
	at io.unthrottled.amii.config.ui.PreferredCharacterTree.<init>(PreferredCharacterTree.kt:299)
	at io.unthrottled.amii.config.ui.PreferredCharacterPanel.<init>(PreferredCharacterPanel.java:11)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.createUIComponents(PluginSettingsUI.java:105)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.$$$setupUI$$$(PluginSettingsUI.java)
	at io.unthrottled.amii.config.ui.PluginSettingsUI.<init>(PluginSettingsUI.java:65)
	... 131 more
Caused by: java.lang.NullPointerException
	at io.unthrottled.amii.assets.VisualEntityRepository.createIndex(VisualEntityRepository.kt:74)
	at io.unthrottled.amii.assets.VisualEntityRepository.updateIndices(VisualEntityRepository.kt:61)
	at io.unthrottled.amii.assets.VisualEntityRepository.access$updateIndices(VisualEntityRepository.kt:9)
	at io.unthrottled.amii.assets.VisualEntityRepository$2.onUpdate(VisualEntityRepository.kt:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:652)
	at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:426)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:401)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:383)
	at com.intellij.util.messages.impl.MessageBusImpl.access$100(MessageBusImpl.java:33)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:189)
	at com.sun.proxy.$Proxy199.onUpdate(Unknown Source)
	at io.unthrottled.amii.assets.RemoteContentManager.initializeAssetCaches(RemoteContentManager.kt:78)
	at io.unthrottled.amii.assets.RemoteContentManager.initializeAssetCaches$default(RemoteContentManager.kt:60)
	at io.unthrottled.amii.assets.RemoteContentManager.<init>(RemoteContentManager.kt:43)
	at io.unthrottled.amii.assets.VisualContentManager.<init>(VisualContentManager.kt:16)
	at io.unthrottled.amii.assets.VisualContentManager.<clinit>(VisualContentManager.kt:15)
	... 154 more
```
</details>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,17 +3,14 @@
 # AMII Changelog
 
 ## [Unreleased]
+
 ### Added
-
-### Changed
-
-### Deprecated
-
-### Removed
+- **Initial Release!!!** Please see the <a href="https://github.com/Unthrottled/AMII#documentation">
+  documentation</a> for more details!
 
 ### Fixed
+- Critical first install startup error
 
-### Security
 ## [0.1.0]
 
 ### Added

--- a/gradle.properties
+++ b/gradle.properties
@@ -3,7 +3,7 @@
 
 pluginGroup = io.unthrottled
 pluginName_ = AMII
-pluginVersion = 0.1.0
+pluginVersion = 0.2.0
 pluginSinceBuild = 201.4515.24
 pluginUntilBuild = 203.*
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl

--- a/src/main/kotlin/io/unthrottled/amii/PluginMaster.kt
+++ b/src/main/kotlin/io/unthrottled/amii/PluginMaster.kt
@@ -1,7 +1,7 @@
 package io.unthrottled.amii
 
 import com.intellij.openapi.Disposable
-import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.ServiceManager
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.project.ProjectManager
 import com.intellij.openapi.project.ProjectManagerListener
@@ -11,32 +11,24 @@ import io.unthrottled.amii.assets.CharacterContentManager
 import io.unthrottled.amii.assets.Status
 import io.unthrottled.amii.assets.VisualContentManager
 import io.unthrottled.amii.listeners.IdleEventListener
-import io.unthrottled.amii.listeners.PLUGIN_UPDATE_TOPIC
-import io.unthrottled.amii.listeners.PluginUpdateListener
 import io.unthrottled.amii.onboarding.UpdateNotification
 import io.unthrottled.amii.onboarding.UserOnBoarding
 import io.unthrottled.amii.platform.LifeCycleManager
 import io.unthrottled.amii.services.WelcomeService
+import io.unthrottled.amii.tools.Logging
 import io.unthrottled.amii.tools.PluginMessageBundle
 import java.util.concurrent.ConcurrentHashMap
 import java.util.concurrent.ConcurrentMap
 import java.util.stream.Stream
 
-internal class PluginMaster :
-  ProjectManagerListener, PluginUpdateListener, Disposable {
+class PluginMaster : ProjectManagerListener, Disposable, Logging {
+
+  companion object {
+    val instance: PluginMaster
+      get() = ServiceManager.getService(PluginMaster::class.java)
+  }
 
   private val projectListeners: ConcurrentMap<String, ProjectListeners> = ConcurrentHashMap()
-  private val messageBusConnection = ApplicationManager.getApplication()
-    .messageBus
-    .connect()
-
-  init {
-    ApplicationManager.getApplication()
-      .invokeLater {
-        messageBusConnection
-          .subscribe(PLUGIN_UPDATE_TOPIC, this)
-      }
-  }
 
   override fun projectOpened(project: Project) {
     registerListenersForProject(project)
@@ -78,10 +70,9 @@ internal class PluginMaster :
   override fun dispose() {
     projectListeners.forEach { (_, listeners) -> listeners.dispose() }
     LifeCycleManager.dispose()
-    messageBusConnection.dispose()
   }
 
-  override fun onUpdate() {
+  fun onUpdate() {
     ProjectManager.getInstance().openProjects
       .forEach { registerListenersForProject(it) }
   }

--- a/src/main/kotlin/io/unthrottled/amii/assets/RemoteContentManager.kt
+++ b/src/main/kotlin/io/unthrottled/amii/assets/RemoteContentManager.kt
@@ -32,8 +32,8 @@ fun interface ContentManagerListener {
 abstract class RemoteContentManager<T : ContentRepresentation, U : Content>(
   private val assetCategory: AssetCategory,
 ) : HasStatus {
-  private lateinit var remoteAndLocalAssets: List<T>
-  private lateinit var localAssets: MutableSet<T>
+  private var remoteAndLocalAssets: List<T> = listOf()
+  private var localAssets: MutableSet<T> = mutableSetOf()
 
   override var status = Status.UNKNOWN
   private val log = Logger.getInstance(this::class.java)

--- a/src/main/kotlin/io/unthrottled/amii/assets/VisualEntityRepository.kt
+++ b/src/main/kotlin/io/unthrottled/amii/assets/VisualEntityRepository.kt
@@ -25,24 +25,30 @@ class VisualEntityRepository : Disposable {
       syncedAssets = 0
     }
 
-    messageBusConnection
-      .subscribe(
-        ContentManagerListener.TOPIC,
-        ContentManagerListener {
-          syncedAssets = syncedAssets or when (it) {
-            AssetCategory.ANIME -> ANIME_SYNC
-            AssetCategory.CHARACTERS -> CHARACTER_SYNC
-            AssetCategory.VISUALS -> VISUAL_SYNC
-            else -> syncedAssets
+    // there is a bit of a circular dependency between
+    // the <code>VisualContentManager</code> and this
+    // class, so we'll just register the update listener
+    // after all of the services initialize.
+    ApplicationManager.getApplication().invokeLater {
+      messageBusConnection
+        .subscribe(
+          ContentManagerListener.TOPIC,
+          ContentManagerListener {
+            syncedAssets = syncedAssets or when (it) {
+              AssetCategory.ANIME -> ANIME_SYNC
+              AssetCategory.CHARACTERS -> CHARACTER_SYNC
+              AssetCategory.VISUALS -> VISUAL_SYNC
+              else -> syncedAssets
+            }
+            if (syncedAssets == ALL_SYNCED) {
+              updateIndices()
+              syncedAssets = 0
+              ApplicationManager.getApplication().messageBus.syncPublisher(SyncedAssetsListener.TOPIC)
+                .onSynced()
+            }
           }
-          if (syncedAssets == ALL_SYNCED) {
-            updateIndices()
-            syncedAssets = 0
-            ApplicationManager.getApplication().messageBus.syncPublisher(SyncedAssetsListener.TOPIC)
-              .onSynced()
-          }
-        }
-      )
+        )
+    }
   }
 
   var visualAssetEntities: Map<String, VisualAssetEntity>

--- a/src/main/kotlin/io/unthrottled/amii/listeners/IDEPluginInstallListener.kt
+++ b/src/main/kotlin/io/unthrottled/amii/listeners/IDEPluginInstallListener.kt
@@ -3,19 +3,11 @@ package io.unthrottled.amii.listeners
 import com.intellij.ide.plugins.DynamicPluginListener
 import com.intellij.ide.plugins.IdeaPluginDescriptor
 import com.intellij.openapi.application.ApplicationManager
-import com.intellij.util.messages.Topic
+import io.unthrottled.amii.PluginMaster
 import io.unthrottled.amii.config.Constants.PLUGIN_ID
+import io.unthrottled.amii.tools.Logging
 
-fun interface PluginUpdateListener {
-  fun onUpdate()
-}
-
-val PLUGIN_UPDATE_TOPIC = Topic.create(
-  "AMII Update",
-  PluginUpdateListener::class.java
-)
-
-class IDEPluginInstallListener : DynamicPluginListener {
+class IDEPluginInstallListener : DynamicPluginListener, Logging {
 
   override fun beforePluginLoaded(pluginDescriptor: IdeaPluginDescriptor) {
   }
@@ -32,9 +24,7 @@ class IDEPluginInstallListener : DynamicPluginListener {
   override fun pluginLoaded(pluginDescriptor: IdeaPluginDescriptor) {
     if (pluginDescriptor.pluginId.idString == PLUGIN_ID) {
       ApplicationManager.getApplication().invokeLater {
-        ApplicationManager.getApplication()
-          .messageBus.syncPublisher(PLUGIN_UPDATE_TOPIC)
-          .onUpdate()
+        PluginMaster.instance.onUpdate()
       }
     }
   }

--- a/src/main/kotlin/io/unthrottled/amii/listeners/ProjectListener.kt
+++ b/src/main/kotlin/io/unthrottled/amii/listeners/ProjectListener.kt
@@ -1,0 +1,18 @@
+package io.unthrottled.amii.listeners
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.ProjectManagerListener
+import io.unthrottled.amii.PluginMaster
+import io.unthrottled.amii.tools.Logging
+
+internal class ProjectListener :
+  ProjectManagerListener, Logging {
+
+  override fun projectOpened(project: Project) {
+    PluginMaster.instance.projectOpened(project)
+  }
+
+  override fun projectClosed(project: Project) {
+    PluginMaster.instance.projectClosed(project)
+  }
+}

--- a/src/main/kotlin/io/unthrottled/amii/tools/BalloonTools.kt
+++ b/src/main/kotlin/io/unthrottled/amii/tools/BalloonTools.kt
@@ -13,7 +13,7 @@ object BalloonTools {
     val frameBounds = ideFrame.component.bounds
     val notificationPosition = RelativePoint(
       ideFrame.component,
-      Point(frameBounds.x + frameBounds.width, NOTIFICATION_Y_OFFSET)
+      Point(frameBounds.x, NOTIFICATION_Y_OFFSET)
     )
     return Pair(ideFrame, notificationPosition)
   }

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -10,6 +10,7 @@
   <extensions defaultExtensionNs="com.intellij">
     <applicationService serviceImplementation="io.unthrottled.amii.config.Config"/>
     <applicationService serviceImplementation="io.unthrottled.amii.services.CharacterGatekeeper"/>
+    <applicationService serviceImplementation="io.unthrottled.amii.PluginMaster"/>
     <applicationService serviceImplementation="io.unthrottled.amii.assets.VisualEntityService"/>
     <applicationService serviceImplementation="io.unthrottled.amii.assets.VisualEntityRepository"/>
     <applicationService serviceImplementation="io.unthrottled.amii.assets.VisualAssetProbabilityService"/>
@@ -28,7 +29,7 @@
               topic="io.unthrottled.amii.events.UserEventListener"/>
     <listener class="io.unthrottled.amii.core.personality.emotions.CoolDownCore"
               topic="io.unthrottled.amii.core.personality.emotions.MoodListener"/>
-    <listener class="io.unthrottled.amii.PluginMaster"
+    <listener class="io.unthrottled.amii.listeners.ProjectListener"
               topic="com.intellij.openapi.project.ProjectManagerListener"/>
     <listener class="io.unthrottled.amii.listeners.IDEPluginInstallListener"
               topic="com.intellij.ide.plugins.DynamicPluginListener"/>

